src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala

```diff
@@ -164,8 +164,8 @@ class Initializer(config: Config) extends SparkSessionWrapper {
       config.setIsPVC(true)
       if (config.debugFlag) println(pvcDetectedMsg)
       logger.log(Level.INFO, pvcDetectedMsg)
-      require(config.workspaceFriendlyName.toLowerCase != config.organizationId.toLowerCase,
-        s"PVC workspaces require the 'workspaceFriendlyName' to be configured in the Overwatch configs for " +
+      require(config.workspaceName.toLowerCase != config.organizationId.toLowerCase,
+        s"PVC workspaces require the 'workspaceName' to be configured in the Overwatch configs for " +
```

Comment:
but it's named `workspace_name` in the RawConfig. Should we use it here? Or you're referring to the field in `Config`?

Reply:
yeah had to change this around, it's working now. I wanted this to be camelCase in the config but in the database tables i wanted it to match the format as all the other columns `workspace_name`...took some doing but finally got it to work. Only down-side is that the OverwatchParams case class has a weird `workspace_name` field in it but it's ok. It caused issues with the upgrades to do it other ways due to state.